src/user/room.ts

```diff
@@ -543,10 +543,10 @@ export class Room extends Accessory implements Sayable {
     }
   }
 
-  public emit (event: 'invite', inviter: Contact,         invitation: RoomInvitation)           : boolean
-  public emit (event: 'leave',  leaverList:   Contact[],  remover?: Contact)                    : boolean
-  public emit (event: 'join',   inviteeList:  Contact[],  inviter:  Contact)                    : boolean
-  public emit (event: 'topic',  topic:        string,     oldTopic: string,   changer: Contact) : boolean
+  public emit (event: 'invite', inviter: Contact,         invitation: RoomInvitation)                       : boolean
+  public emit (event: 'leave',  time: Date, leaverList:   Contact[],  remover?: Contact)                    : boolean
```

Comment:
Please move all the `time` argument to the end of the argument lists, and make it optional, so that it will not become a breaking change.

Reply:
And also, please change the `remover?` argument for the `emit` function to `remover`. I think it's better to not optional as a emit argument.

Reply:
Will do.